### PR TITLE
Remove the extra proxy sidecar

### PR DESCRIPTION
**Pull request by vatsalparekh**
Signed-off-by: Vatsal Parekh <vparekh@redhat.com>

**What this PR does / why we need it**:
Remove the extra proxy sidecar
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -9,16 +9,6 @@ spec:
   template:
     spec:
       containers:
-      - name: kube-rbac-proxy
```
Comment (omeryahud) [inline config/default/manager_auth_proxy_patch.yaml:12]: I think commenting out the entire patch is a better idea https://github.com/kubevirt/ssp-operator/blob/master/config/default/kustomization.yaml#L31

Comment (vatsalparekh) [reply config/default/manager_auth_proxy_patch.yaml:12]: But we still need the manager container here

Comment (omeryahud) [reply config/default/manager_auth_proxy_patch.yaml:12]: The manager container will remain as it is present in the deployment base resource https://github.com/kubevirt/ssp-operator/blob/master/config/manager/manager.yaml#L25

File: config/default/manager_auth_proxy_patch.yaml [hunk 2]
```diff
@@ -3,23 +3,23 @@
 apiVersion: apps/v1
```
Comment (omeryahud) [inline config/default/manager_auth_proxy_patch.yaml:3]: Please fix indentation in this file and I think that's it

Comment (vatsalparekh) [reply config/default/manager_auth_proxy_patch.yaml:3]: Corrected

**Pull request merged**